### ISSUE TITLE
Make sure check for presence of maintainer check works

### ISSUE THREAD

**Issue by TomasTomecek** [user-cont/colin#130]
We've got a report internally that $subject does not work. Please investigate and ideally create a test case for this.

    I am pretty sure the "WARN:Label 'maintainer' has to be specified." is a
    false warning, unless I am blind and can not see the typo in my Dockerfile.

CC @fkluknav

**Comment by fkluknav** [comment]
`LABEL maintainer="Red Hat, Inc."`
causes
`WARN:Label 'maintainer' has to be specified.`

**Comment by TomasTomecek** [comment]
Yeah, our regex is `@redhat.com$`. This is worth discussing internally.

**Comment by fkluknav** [comment]
What is the standard for 'maintainer' label in images from Red Hat? Is there any? A person? A group? A company?

**Comment by TomasTomecek** [comment]
Exactly, probably there is none. Therefore this should be brought up internally so we all agree on one standard.

@fkluknav Would you like to bring it up or should we take care of it?
